content/introduction-and-setup/walkthrough/creating-a-project/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Creating a C# Project"
+date: 2022-12-15T09:16:07-06:00
+draft: false
+weight: 4
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Following the “Hello World” trend, let’s create a new Visual Studio project.
+
+1. Create a new folder to hold your C# practice files. Since you will be creating lots of small projects as you move through this course, we suggest that you also add sub-folders with names corresponding to the related chapters and projects. Something like `csharp-practice/chapter-name/project-name`.
+1. In Visual Studio, from the project opener window, select the option to create a *New Visual Studio project*.
```

Comment:
In the main pop up Window for Mac it is just  *+New* from the file menu *New Project*

Reply:
Removed italics so it is clearer for both Windows and Mac users.